docs/example/workspace.dsl

```diff
@@ -156,10 +156,10 @@ workspace "Big Bank plc" "This is an example workspace to illustrate the key fea
     views {
         properties {
             "c4plantuml.elementProperties" "true"
-            "structurizr.style.colors.primary" "#485fc7"
-            "structurizr.style.colors.secondary" "#ffffff"
-            "structurizr.style.favicon.path" "site/favicon.ico"
-            "structurizr.style.logo.path" "site/logo.png"
+            "site-generatr.style.colors.primary" "#485fc7"
+            "site-generatr.style.colors.secondary" "#ffffff"
+            "site-generatr.style.favicon.path" "site/favicon.ico"
```

Comment:
For consistency I would prefer `siteGeneratr.style.faviconPath` since it fits the style of structurizr properties e.g. `c4plantuml.elementProperties`.
That said, `site-generatr` looks imho much beter that `siteGeneratr`. Well, may be just `generatr` isn't such a bad idea after all ;)

Reply:
Hm, fair point indeed... While I agree that kebab case looks better for properties, I think that consistency is more important. So lowerCamelCase it is, I guess 😄.

Reply:
Not really... I guess I slightly prefer `generatr`, because it's shorter.

Reply:
Ack, I've adjusted this PR.